can/generated_2021/uwrt_mars_rover_can.dbc

```diff
@@ -426,19 +426,43 @@ CM_ SG_ 1826 COMMON_debugSignal2 "CAN debug signal";
 CM_ BO_ 1827 "CAN message for debugging or placeholder purposes";
 CM_ SG_ 1827 COMMON_debugSignal3 "CAN debug signal";
 BA_DEF_ SG_  "SystemSignalLongSymbol" STRING ;
+BA_DEF_ BO_  "GenMsgCycleTime" INT 0 65535;
 BA_DEF_DEF_  "SystemSignalLongSymbol" "";
+BA_DEF_DEF_  "GenMsgCycleTime" 0;
+BA_ "GenMsgCycleTime" BO_ 1570 0.01;

```

Comment:
probs not related to this pr, but why weren't these cycle times generated before?

@cindyli-13 ?

Reply:
Hmm no idea, what do these mean?

Reply:
They define the cycle times of each signal and msg